test/src/test/java/hudson/model/RunMapTest.java

```diff
@@ -43,7 +44,7 @@
         FreeStyleBuild b1 = r.assertBuildStatusSuccess(p.scheduleBuild2(0));
 
         // now create a build that hangs until we signal the OneShotEvent
-        p.getBuildersList().add(new SleepBuilder(9999999));
+        p.getBuildersList().add(new SleepBuilder(Long.MAX_VALUE));
```

Comment:
Using `Long.MAX_VALUE` to be consistent with other tests and to more clearly indicate the intention that this is meant to run "forever" before being interrupted.